packages/babel-plugin-lingui-macro/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@lingui/babel-plugin-lingui-macro",
```

Comment:
Please also add the `publishConfig`:

```json
"publishConfig": {
  "access": "public"
},
```

Otherwise, it will fail during the first publish by CI/CD.